app/src/forms/email/emailService.js

```diff
@@ -287,8 +287,9 @@ const service = {
   submissionReceived: async (formId, submissionId, body, referer) => {
     try {
       const { configData, contexts } = await buildEmailTemplate(formId, submissionId, EmailTypes.SUBMISSION_RECEIVED, referer, { body });
-
-      return service._sendEmailTemplate(configData, contexts);
+      if (contexts[0].to.length > 0) {
```

Comment:
Use shorthand truthiness check:
```js
if (contexts[0].to.length) {
```

Reply:
I'm not sure this should be based on checking contexts in the email service. Aren't the contexts set by the supplied email address in the body, so shouldn't it check a level or 2 higher to even see if email service should be called at all?

So check body email address at the controller level. The code flow is really "do I need to do anything about an email here?" when controlling the submission POST

Reply:
IE, controller.createSubmission line 126